packages/node_modules/@node-red/nodes/core/function/89-trigger.js

```diff
@@ -78,10 +78,10 @@ module.exports = function(RED) {
 
         var pendingMessages = [];
```

Comment:
For the Trigger node, can I check  *when* `done` is called for a message?

From what I think I see, it calls `done` when the message has been received and:
 - the 'first' message has been sent,
 - any timer has been set up for the second message

But it does *not* wait for the second message to be sent before calling done?

I think that's okay - but want to check I'm not missing something.

Reply:
Yes, your understanding is what I intended to do in this code. 
But, to make consistent with other node (e.g. delay node), it might be better to suspend `done` while there is something in queue or interval timer caused by the message.

What do you think about this?  If you prefer the second behaviour, I'll update the code.

Reply:
I think, current implementation is enough because input message can be seen as trigger, so when triggering (set a timer, block, etc.) is done, we can see it as a completion of the message.